benchmarks/htap/schemas/s64da/schema.sql

```diff
@@ -0,0 +1,153 @@
+CREATE TABLE IF NOT EXISTS warehouse (
```

Comment:
For the other benchmarks we have mmultiple schemas: psql_native, s64da_native, ..._native_enhanced, ...._performance, ...
For htap we have just psql_native and s64da. What is the reasoning behind this?

Reply:
We do have psql_native as well. But good point, I will align the names with the other benchmarks and also structure them the same way.